src/views/mfa-verify/inputs/ShowPasswordBox.js

```diff
@@ -1,50 +0,0 @@
-/*!
```

Comment:
Removed`src/views/mfa-verify/inputs/ShowPasswordBox.js` as this is a courage component now https://github.com/okta/courage/blob/master/src/views/forms/inputs/PasswordBox.js